cmd/argo/commands/cron/create.go

```diff
@@ -48,7 +48,10 @@ func NewCreateCommand() *cobra.Command {
 
 func CreateCronWorkflows(filePaths []string, cliOpts *cliCreateOpts, submitOpts *wfv1.SubmitOpts) {
 	ctx, apiClient := client.NewAPIClient()
-	serviceClient := apiClient.NewCronWorkflowServiceClient()
+	serviceClient, err := apiClient.NewCronWorkflowServiceClient()
+	if err != nil {
```

Comment:
minor: prefer `errors.CheckError(err)`

Reply:
I don't feel strongly. Any comment prefixed with "minor" is really a suggestion.